### PR TITLE
Solved Issue #3992

### PR DESCRIPTION
**Pull request by joshuazeltser**
Fixed issue #3992. Added appropirate title to the icon and other aria tags added to make dropdowns more understandable when read by a narrator

### CODE REVIEW

File: notebook/static/base/js/namespace.js [hunk 1]
```diff
@@ -1,83 +1,83 @@
-// Copyright (c) Jupyter Development Team.
-// Distributed under the terms of the Modified BSD License.
-
-
```
Comment (takluyver) [inline notebook/static/base/js/namespace.js]: Why is this entire file showing up as changed? Can you do it without this?

**Comment by takluyver** [comment]
Comments from #4695 also apply here.

File: notebook/static/base/js/namespace.js (hunk 1 above)
Comment (joshuazeltser) [reply notebook/static/base/js/namespace.js]: As far as I can see there are no changes. I have no idea why this has happened

File: notebook/static/tree/js/notebooklist.js [hunk 2]
```diff
@@ -386,6 +386,7 @@ define([
         var root = $('<li/>').append(
             $("<a/>")
             .attr('href', root_url)
+            .attr('title','Link to root folder'+root_url)
```
Comment (takluyver) [inline notebook/static/tree/js/notebooklist.js]: This presumably needs integrating with the translation machinery.

Also, it doesn't seem right to put the URL in the title. Again, I'm quite ignorant of accessibility concerns, but presumably screenreader tools have a way to inspect the URL of a link if the user wants that.

Comment (joshuazeltser) [reply notebook/static/tree/js/notebooklist.js]: This has been updated

**Comment by takluyver** [comment]
I've cherry-picked the intended changes from here as PR #4719, preserving commit authorship, so I'll close this one.

**Pull request closed without merging**